deepmd/pt/model/model/sezm_model.py

```
@@ -881,6 +876,63 @@ def forward(
             model_predict["updated_coord"] += coord
         return model_predict
 
+    def forward_embedding(
+        self,
+        coord: Float[Tensor, "nf nloc 3"] | Float[Tensor, "nf nloc_x3"],
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)